src/popover/popover.ts

```diff
@@ -27,7 +27,11 @@ let nextId = 0;
 @Component({
   selector: 'ngb-popover-window',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  host: {'[class]': '"popover bs-popover-" + placement', 'role': 'tooltip', '[id]': 'id'},
+  host: {
+    '[class]': '"popover bs-popover-" + placement.split("-")[0]+" bs-popover-" + placement',
```

Comment:
This, for cases like `top-right`, would add the following CSS classes: `bs-popover-top` and `bs-popover-top-right`. Is this our intention? If so, why?

Reply:
Yes, You are right the intention is to add a new css-class e.g. bs-popover-top-right to extend the css rules defined by bs-popover-top css-class of bootstrap. The new classes are defined in the styles e.g.
styles: [`
    :host.bs-popover-top .arrow, :host.bs-popover-bottom .arrow {
      left: 50%;
    } 
    ...
  `].

This is same for tooltip.

Reply:
So, I think that there is a bit of the issue here since for "simple" positioning options (like `top`) we would have `bs-popover-top` class twice here. Could you think of a way of avoiding this?